### ISSUE TITLE
Hyperparameter tuning and Ray for paddleseg

### ISSUE THREAD

**Issue by Mimicvat** [PaddlePaddle/PaddleSeg#3565]
### 问题确认 Search before asking

- [X] 我已经搜索过问题，但是没有找到解答。I have searched the question and found no related answer.

### 请提出你的问题 Please ask your question

Hi, I would like to know if there are scripts or libraries that do AutoML, similar to PyCaret and Auto-sklearn, on top of paddleseg? I would like to do a batch of scientific experiments changing the loss parameters, testing various models and augmentations and then save all the results and weights in a systematic way so that I can analyze them later. From what I've researched, there's a similar project for mmsegmentation [https://github.com/SIAnalytics/siatune](https://github.com/SIAnalytics/siatune)

Or some easy way to integrate Ray into paddleseg [https://docs.ray.io/en/latest/tune/](https://docs.ray.io/en/latest/tune/)

**Comment by shiyutang** [comment]
We have not yet integrate this feature, you can help us add it.
